docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2227,19 +2227,24 @@ definitions:
         example: 'pennybags.payment-business-event'
       occurred_at:
         description: |
-          Timestamp of when the event was created by the producer application. 
-          (Typically, it differs from when the event is posted by the producer and received by Nakadi via the API.)
+         Technical timestamp of when the event object was created during processing of the
+         business event by the producer application. Note, it may differ from the timestamp 
+         when the related real-world business event happened (e.g. when the packet was handed 
+         over to the customer), which should be passed separately via an event type specific 
+         attribute. 
+         Depending on the producer implementation, the timestamp is typically some milliseconds  
+         earlier than when the event is published and received by the API event post endpoint 
+         server -- see below.
         type: string
         format: RFC 3339 date-time
         example: '1996-12-19T16:39:57-08:00'
       received_at:
         type: string
         readOnly: true
-        description: |
-          Timestamp of when the event was received by Nakadi via the event post endpoints. 
-          It is automatically enriched, and events will be rejected if set by the Event producer.
-        format: RFC 3339 date-time
-        example: '1996-12-19T16:39:57-08:00'
+      description: |
+        Timestamp of when the event was received via the API event post endpoints. 
+        It is automatically enriched, and events will be rejected if set by the event producer.
+      example: '1996-12-19T16:39:57-08:00'
```

Comment:
@tfrauenstein Seems that this is not the intended change (move 2 spaces left the description of `received_at`

Reply:
@antban yes, this minor change is intended  (lower case of 'event producer').

Reply:
Github does not allow me to add a multi-line suggestion to the lines that already have comments.  I think @antban is pointing out that `description` and `example` were unintentionally moved two spaces to the left, so they are no longer belonging to `received_at` description.  Also not sure if removal of `format` was intentional.

Reply:
+1 -- fixed: indention and format definition